### PR TITLE
Support python3 and update dependencies

### PR DESCRIPTION
**Pull request by jessemyers**
Plus a few cosmetic changes.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/.dockerignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 *
 !{{cookiecutter.project_name}}
-!manage.py
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/.dockerignore]: This file isn't part of newer services.

File: {{cookiecutter.repo_name}}/circle.yml [hunk 2]
```diff
@@ -1,7 +1,3 @@
-general:
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/circle.yml]: There's no substantive change here; I've just updated the ordering to match recent services.

File: {{cookiecutter.repo_name}}/setup.py [hunk 3]
```diff
@@ -15,17 +15,16 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/setup.py]: I'm just catching up the versions here with latest releases.

File: {{cookiecutter.repo_name}}/tox.ini [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, lint
+envlist = py27, py35, lint
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/tox.ini]: Enable Python3 builds!

File: {{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/tests/routes/test_example_routes.py [hunk 5]
```diff
@@ -71,7 +71,7 @@ def test_search(self):
 
         with self.graph.app.test_request_context():
             assert_that(response.status_code, is_(equal_to(200)))
-            data = loads(response.data)
+            data = loads(response.data.decode("utf-8"))
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/tests/routes/test_example_routes.py]: `response.data` are `bytes` in Python3; need to convert to string to pass to `loads`.

**Comment by adamhadani** [comment]
LGTM

**Pull request closed without merging**